3회차/우상민/2_암호생성기.py

```diff
@@ -1,3 +1,23 @@
 import sys
 
 sys.stdin = open("_암호생성기.txt")
+
+for C in range(1, 11): 
+    T = int(input())
+    matrix_2 = input().split()
+    count = 0
+    while int(matrix_2[7]) > 0:
+        count = 0
+        for i in range(5):
+            count += 1
+            N = matrix_2[0]
+            matrix_2.remove(N)
```

Comment:
```suggestion
            matrix_2.pop(0)
```
리스트에서 앞의 원소를 빼는 경우엔 `pop(0)`를 사용해주시는 게 좋아요!